docs/howto/create_new_package.md

```diff
@@ -0,0 +1,40 @@
+# HOWTO: Create new package/data stream
+
+## Introduction
+
+Elastic Packages can be used to bootstrap a new package or add a data stream using an embedded archetype ([resource templates](https://github.com/elastic/elastic-package/tree/master/internal/packages/archetype)).
+It's advised to use `elastic-package create` to build new package rather than copying sources of an existing package.
+With archetype you can be sure that you're following latest recommendations for the package format.
+
+### Create new package
+
+Prerequisites:
+
+_Pick the directory where you'd like to create a new package. For integrations, it's: [packages/](https://github.com/elastic/integrations/tree/master/packages)._
+
+Steps:
+
+1. Bootstrap new package using the TUI wizard: `elastic-package create package`.
+2. Fine-tune the created package:
```

Comment:
These are manual steps right? Maybe mention it so it's extra clear?

Reply:
Fixed, good idea